tox.ini

```diff
@@ -47,7 +47,7 @@ directory = coverage-html
 # end coverage configuration
 
 [flake8]
-max-line-length = 105
```

Comment:
Please, do not change flake8 configuration to conform to your PR, do the opposite instead